interface/src/app/gene/gene-network.tpl.html

```diff
@@ -0,0 +1,10 @@
+<div>
+  <h1 class="page-header">
+    Gene Network
+  </h1>
+  <div class="well">
+    <geneset-search-form ng-if="userObj"></geneset-search-form>
```

Comment:
This isn't exactly a code issue, so it's up to you to decide if it's an issue worth addressing right now, later via an issue or maybe it's not actually an issue at all. I meant to be just asking the question while I had it on my mind. Until I see the actual screen in action, I can't give solid design review advice.